src/server/src/db/queries/timeQueries/getCategoryTimeOverview.sql

```diff
@@ -0,0 +1,6 @@
+SELECT category, count() AS minutes
```

Comment:
I think these are simple enough to do with knex and instead of using sub select let's use a with statement